src/plugins/expressions/common/expression_renderers/types.ts

```diff
@@ -99,4 +100,6 @@ export interface IInterpreterRenderHandlers {
    * Downstream consumers of the uiState handler will need to cast for now.
    */
   uiState?: unknown;
+
+  getExecutionContext: () => KibanaExecutionContext | undefined;
```

Comment:
This is slightly incorrect as we should not block the LSP when extending this interface.

```suggestion
  getExecutionContext(): KibanaExecutionContext | undefined;
```